packages/serverless-nextjs-component/examples/multiple-instance-environment/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "multiple-instance-environment",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "deployStaging": "STAGE=staging serverless",
```

Comment:
Could we change this so it is `serverless --env=staging` and same for prod below. This looks to be the official recommendation here: https://github.com/serverless/template#programmatic-usage-and-custom-environments

Reply:
Will do.